app/frontend/stores/session-store.ts

```diff
@@ -6,7 +6,7 @@ export const SessionStoreModel = types
   .model("SessionStoreModel")
   .props({
     loggedIn: types.optional(types.boolean, false),
-    isValidating: types.optional(types.boolean, false),
+    isValidating: types.optional(types.boolean, true),
```

Comment:
how come this is true by default?

Reply:
@sunto this true by default to prevent race conditions.

Reply:
@sunto can confirm, tawshif and I worked this out.
basically we dont want to show the logged in pages until we are actually sure the user is logged in, which depends on the isValidating flags. Leaving this as default false means that the useEffect hooks in logged in versions of the same route will run and then again on the logged in page.